pkg/permissions/rule.go

```diff
@@ -99,12 +99,18 @@ func (r Rule) ValuesValid(o Validable) bool {
 	return false
 }
 
-// ValuesContain returns true if the value is in r.Values
-func (r Rule) ValuesContain(value string) bool {
-	for _, v := range r.Values {
-		if v == value {
-			return true
+// ValuesContain returns true if all the values are in r.Values
+func (r Rule) ValuesContain(values ...string) bool {
+	for _, value := range values {
+		valueOK := false
+		for _, v := range r.Values {
+			if v == value {
+				valueOK = true
```

Comment:
nit: a little suboptimal... even if it is not critical for now, it may be worth to add a `TODO(optim)` since the number of values in a rule is a user input without a hard limit.